examples/run_rbfe_legs.py

```diff
@@ -0,0 +1,291 @@
+import pickle
+from argparse import ArgumentParser
+from datetime import datetime
+from pathlib import Path
+
+import jax
+
+# Enable 64 bit jax
+jax.config.update("jax_enable_x64", True)
+
+
+import numpy as np
+from numpy.typing import NDArray
+from rdkit import Chem
+
+# This is needed for pickled mols to preserve their properties
+Chem.SetDefaultPickleProperties(Chem.PropertyPickleOptions.AllProps)
+
+from timemachine.constants import DEFAULT_ATOM_MAPPING_KWARGS, DEFAULT_FF
+from timemachine.fe import atom_mapping
+from timemachine.fe.free_energy import HREXParams, RESTParams, WaterSamplingParams
+from timemachine.fe.rbfe import (
+    DEFAULT_NUM_WINDOWS,
+    HREXSimulationResult,
+    MDParams,
+    run_complex,
+    run_solvent,
+    run_vacuum,
+)
+from timemachine.fe.utils import get_mol_name, plot_atom_mapping_grid, read_sdf_mols_by_name
+from timemachine.ff import Forcefield
+from timemachine.md.exchange.utils import get_radius_of_mol_pair
+from timemachine.parallel.client import CUDAPoolClient, FileClient
+from timemachine.parallel.utils import get_gpu_count
+
+
+def run_leg(
+    file_client: FileClient,
+    leg_name: str,
+    mol_a: Chem.Mol,
+    mol_b: Chem.Mol,
+    core: NDArray,
```

Comment:
tiny nit / style opinion: the ordering `mol_a`, `mol_b`, `core`, `leg_name` seems more natural to me (i.e. `(*edge, leg_name)`)